tests/lib/Key.spec.js

```diff
@@ -32,16 +33,16 @@ describe('Key', () => {
     });
 
     it('can derive addresses (LEGACY)', () => {
-        const key = new Key(Dummy.keys[0], Key.Type.LEGACY);
+        const key = new Key(Dummy.secrets[0]);
         const address = 'NQ71 CT4K 7R9R EHSB 7HY9 TSTP XNRQ L2RK 8U4U';
         expect(key.deriveAddress('m').toUserFriendlyAddress()).toEqual(address);
         expect(key.deriveAddress('m/0\'').toUserFriendlyAddress()).toEqual(address);
     });
 
     it('can derive addresses (BIP39)', () => {
-        const key = new Key(Dummy.keys[0], Key.Type.BIP39);
-        const address1 = 'NQ55 07AD F3U4 9FTD XP5N 1UMS 2FVT 3N69 Q0H9';
-        const address2 = 'NQ15 YMBC GHHD S4DY CPR5 LC74 HAKD ES3R 5B7P';
```

Comment:
I had to update the addresses here, because we were using the `Dummy.keys[0]` for derivation here, which is supposed to be a legacy single-address wallet. Now, this doesn't work anymore because `Key` detects the type of secret and would derive the same legacy address for both paths, as in the test above.